app/views/content/steps-to-become-a-teacher/_step_3_consider_getting_experience.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Experiencing life in a school and a classroom can help to decide if teaching is right for you and who you want to teach.</p>
```

Comment:
```suggestion
<p>Experiencing life in a school can help you decide if teaching is right for you and who you want to teach. This is a good way to give you a taste of what the classroom is really like</p>
```
Just wondering if we want to add another lil sentence here, just to further promote why getting school experience can be good?